addons/web_editor/static/src/xml/snippets.xml

```diff
@@ -168,15 +169,17 @@
                             <span t-esc="category.text"/>
                         </div>
                         <div t-att-id="category.id === 'snippet_custom' ? 'snippet_custom_body' : ''" class="o_panel_body" t-on-pointerup="_onMouseUp">
-                            <t t-foreach="category_value" t-as="snippet" t-key="snippet.id">
+                            <t t-foreach="category_value" t-as="snippet" t-key="snippet.key">
                                 <div t-if="snippet.visible"
                                     class="oe_snippet"
                                     t-att-class="{ 'o_disabled': snippet.disabled, 'o_snippet_install': snippet.installable, 'o_we_draggable': !snippet.renaming and !snippet.installable and !snippet.disabled }"
                                     t-att-name="snippet.displayName"
                                     t-att-data-oe-snippet-id="snippet.id"
                                     t-att-data-module-id="snippet.moduleId"
-                                    t-on-click="_onSnippetClick"
-                                    t-att-data-tooltip="snippet.disabled ? disabledTooltip : false">
+                                    t-on-click="(ev) => this._onSnippetClick(ev.currentTarget)"
```

Comment:
Is there a reason to use `ev.currentTarget` as a param for `_onSnippetClick` rather than `ev` (which wouldn't need an arrow function here)? It doesn't look like it to me. (And it would make more sense considering the name of the function if `ev` were the parameter.)

Reply:
I used `ev.currentTarget` as parameter to avoid to read on `ev` 4 times to get the current target in the function.
But doesn't matter, I changed that... Arrow function is needed anyway to avoid a traceback

Reply:
My bad, you are right (no traceback actually without the arrow function) I removed it